auto3dseg/algorithm_templates/segresnet2d/scripts/utils.py

```diff
@@ -1,9 +1,54 @@
 import numpy as np
 import torch
 
+import logging
 from monai.apps.auto3dseg.auto_runner import logger
 print = logger.debug
 
+def logger_configure(log_output_file: str = None, debug = False, global_rank=0) -> None:
+
+    log_config = {
+        "version": 1,
+        "disable_existing_loggers": False,
+        "formatters": {"monai_default": {"format": "%(message)s"}},
+        "loggers": {
+            "monai.apps.auto3dseg.auto_runner": {"handlers": ["console", "file"], "level": "DEBUG", "propagate": False}
+        },
+        # "filters": {"rank_filter": {"{}": "__main__.RankFilter"}},
+        "handlers": {
+            "console": {
+                "class": "logging.StreamHandler",
+                "level": "INFO",
+                "formatter": "monai_default",
+                # "filters": ["rank_filter"],
+            },
+             "file": {
+                "class": "logging.FileHandler",
+                "filename": "runner.log",
+                "mode": "a",
+                "level": "DEBUG",
+                "formatter": "monai_default",
+                # "filters": ["rank_filter"],
+            },
+        },
+    }
+
+    if log_output_file is not None:
+        log_config["handlers"]["file"]["filename"] = log_output_file
+        log_config["handlers"]["file"]["level"] = "DEBUG"
+    else:
+        log_config["handlers"]["file"]["level"] = "CRITICAL"
+
+    if debug or bool(os.environ.get("SEGRESNET_DEBUG", False)):
```

Comment:
this requires import os
======================================================================
ERROR: test_autorunner (__main__.TestAutoRunner)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/__w/research-contributions/research-contributions/core/tests/test_integration_autorunner.py", line 124, in test_autorunner
    runner.run()
  File "/__w/research-contributions/research-contributions/core/monai/apps/auto3dseg/auto_runner.py", line 775, in run
    bundle_generator.generate(self.work_dir, num_fold=self.num_fold, allow_skip=self.allow_skip)
  File "/__w/research-contributions/research-contributions/core/monai/apps/auto3dseg/bundle_gen.py", line 589, in generate
    gen_algo.export_to_disk(output_folder, name, fold=f_id)
  File "/tmp/tmpcqeax5go/work_dir/algorithm_templates/segresnet2d/scripts/algo.py", line 315, in export_to_disk
  File "/__w/research-contributions/research-contributions/core/monai/apps/auto3dseg/bundle_gen.py", line 161, in export_to_disk
    self.fill_records = self.fill_template_config(self.data_stats_files, self.output_path, **kwargs)
  File "/tmp/tmpcqeax5go/work_dir/algorithm_templates/segresnet2d/scripts/algo.py", line 84, in fill_template_config
  File "/tmp/tmpcqeax5go/work_dir/algorithm_templates/segresnet2d/scripts/utils.py", line 42, in logger_configure
NameError: name 'os' is not defined

Reply:
you're right. I just fixed it